i18n/ja/docusaurus-plugin-content-docs/current/topics/develop/sdks/swift/cheat-sheet.mdx

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 1
+sidebar_label: Topics Cheat Sheet
+sidebar_class_name: sidebar-item-swift
+title: Topics cheat sheet for Swift
+description: Get started really quickly coding using Swift with Momento Topics
+---
+
+import { SdkExampleCodeBlock } from "@site/src/components/SdkExampleCodeBlock";
+// This import is necessary even though it looks like it's un-used; The inject-example-code-snippet
+// plugin will transform instances of SdkExampleCodeBlock to SdkExampleCodeBlockImpl
+import { SdkExampleCodeBlockImpl } from "@site/src/components/SdkExampleCodeBlockImpl";
+
+# Cheat Sheet for Swift with Momento Topics
+
+If you need to get going quickly with Swift and Momento Topics, this page contains the basic API calls you'll need. [Check the Swift SDK examples](https://github.com/momentohq/client-sdk-swift/tree/main/Examples) for complete, working code samples.
+
+## Install the Momento SDK
+
+The latest version of the Momento Swift SDK is available on GitHub: [momentohq/client-sdk-swift](https://github.com/momentohq/client-sdk-swift).
+
+## Get your Momento API key
+
+You'll need a Momento API key. You can get one from the [Momento Web Console](https://console.gomomento.com/).
+Once you have your token, store it in an environment variable so that the Momento client can consume it:
+
+```
+export MOMENTO_API_KEY=<your api key here>
+```
+
+**Note**: it is best practice to put the API key into something like AWS Secret Manager or GCP Secret Manager instead of an environment variable for enhanced security, but we are using one here for demo purposes.
+
+## Set up a TopicClient
+This code creates the TopicClient that you will use to interact with your pub/sub topic.
+
+<SdkExampleCodeBlock language={'swift'} snippetId={'API_InstantiateTopicClient'} />
+
+## Publish a message to a topic
+This is an example of publishing a message to a topic called "topic", then catching the return value to check if the publish was successful.
+
+<SdkExampleCodeBlock language={'swift'} snippetId={'API_TopicPublish'} />
+
+## Subscribe to a topic
+This is an example of subscribing to a topic called "topic", then catching the return value to check if it was successful. If a subscription was received, a `for try await` loop is used to asynchronously receive and print the messages that are published to this topic.
+
+<SdkExampleCodeBlock language={'swift'} snippetId={'API_TopicSubscribe'} />
```

Comment:
These are showing up blank for me. Looks like they need to be `API_Publish` and `API_Subscribe` instead,

Reply:
Ah, nevermind, I see the fix for this in the other PR.